src/Host.h

```diff
@@ -56,16 +56,13 @@ class dlgNotepad;
 class TMap;
 
 
-class Host  : public QObject
-{
+class Host : public QObject {
     friend class XMLexport;
     friend class XMLimport;
 
 public:
-
-                       Host( int port, const QString& mHostName, const QString& login, const QString& pass, int host_id );
-
-                       ~Host();
+    Host(int port, const QString& mHostName, const QString& login, const QString& pass, int host_id);
```

Comment:
... 💡and should we be so minded to sort them, it is much harder to do so like this!